jena-arq/src/main/java/org/apache/jena/sparql/expr/nodevalue/XSDFuncOp.java

```diff
@@ -1171,12 +1168,12 @@ public static int compareDateTimeFO(NodeValue nv1, NodeValue nv2) {
         return x ;
     }
 
-    /** Return a normalized XMLGregorianCalendar appropriate for comparision */
+    /** Return a normalized XMLGregorianCalendar appropriate for F&O comparision using teh XSD algorithm. */
```

Comment:
s/comparision/comparison (not added in this PR, but worth fixing?). Also s/teh/the